Kickstarter-iOS/Tests/Views/LiveStreamDiscoveryViewControllerTests.swift

```diff
@@ -28,6 +28,48 @@ internal final class LiveStreamDiscoveryViewControllerTests: TestCase {
       |> LiveStreamEvent.lens.liveNow .~ true
       |> LiveStreamEvent.lens.name .~ "Comin‘ to you live!"
       |> LiveStreamEvent.lens.creator.name .~ "A creator with a really long name to wrap onto two lines"
+      |> LiveStreamEvent.lens.numberPeopleWatching .~ 1428
+
+    let futureEvent = .template
+      |> LiveStreamEvent.lens.startDate .~ MockDate().addingTimeInterval(future).date
+      |> LiveStreamEvent.lens.liveNow .~ false
+      |> LiveStreamEvent.lens.name .~ "Future live event!"
+    let pastEvent = .template
+      |> LiveStreamEvent.lens.startDate .~ MockDate().addingTimeInterval(-60 * 60 * 24).date
+      |> LiveStreamEvent.lens.liveNow .~ false
+      |> LiveStreamEvent.lens.hasReplay .~ true
+      |> LiveStreamEvent.lens.replayUrl .~ "http://www.replay.com"
+      |> LiveStreamEvent.lens.name .~ "We were live, but we aren‘t anymore. Check out our replay!"
+
+    let events = [futureEvent, liveEvent, pastEvent]
+
+    let liveStreamService = MockLiveStreamService(fetchEventsResult: .success(events))
+
+    combos(Language.allLanguages, [Device.phone4_7inch, .pad]).forEach { language, device in
+      withEnvironment(language: language, liveStreamService: liveStreamService) {
+
+        let vc = Storyboard.LiveStreamDiscovery.instantiate(LiveStreamDiscoveryViewController.self)
+        let (parent, _) = traitControllers(device: device, orientation: .portrait, child: vc)
+        vc.isActive(true)
+        parent.view.frame.size.height = device == .pad ? 1_500 : 1_100
+
+        self.scheduler.advance()
+
+        FBSnapshotVerifyView(parent.view, identifier: "lang_\(language)_device_\(device)")
+      }
+    }
+  }
+
+  func testView_NumberPeopleWatchingHidden() {
```

Comment:
i think we could minimize the number of screens for this test (since most of it is covered in the other tests) by just testing english and on one device. that gets us the coverage of checking the count hides when it should.